sky/utils/command_runner.py

```diff
@@ -36,8 +35,9 @@ def _ssh_control_path(ssh_control_filename: Optional[str]) -> Optional[str]:
     """Returns a temporary path to be used as the ssh control path."""
     if ssh_control_filename is None:
         return None
-    username = getpass.getuser()
-    path = (f'/tmp/skypilot_ssh_{username}/{ssh_control_filename}')
+    base_path = '/tmp/skypilot_ssh_'
+    user_hash = common_utils.get_user_hash()
+    path = os.path.join(base_path, f'{user_hash}/{ssh_control_filename}')
```

Comment:
How about we keep the previous naming format for the folder? Reason: on a multi-tenant system, if a user creates a folder /tmp/skypilot_ssh`, the other users will not be able to create new files/folders under that folder
```suggestion
    user_hash = common_utils.get_user_hash()
    path = f'/tmp/skypilot_ssh_{user_hash}/{ssh_control_filename}'
```

Reply:
Ok, great,  I agree with you, your suggestion makes more sense